backend/package.json

```diff
@@ -26,10 +26,6 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "dotenv -c -- jest --config ./test/jest-e2e.json",
     "test:e2e:cov": "dotenv -c -- jest --config ./test/jest-e2e.json --coverage",
-    "scratch:download:types:definitions": "curl -o .temp/scratch/sb3_definitions.json https://raw.githubusercontent.com/scratchfoundation/scratch-parser/refs/heads/master/lib/sb3_definitions.json",
-    "scratch:download:types:schema": "curl -o .temp/scratch/sb3_schema.json https://raw.githubusercontent.com/scratchfoundation/scratch-parser/refs/heads/master/lib/sb3_schema.json",
-    "scratch:download:types": "yarn mkdirp .temp/scratch && yarn scratch:download:types:definitions && yarn scratch:download:types:schema",
-    "scratch:update-types": "yarn scratch:download:types && cd .temp/scratch && json2ts -i sb3_schema.json -o ../../src/ast/types/input/scratch/generated/sb3.ts",
     "sentry:sourcemaps": "sentry-cli sourcemaps inject --org hep-vaud --project crt-backend ./dist && sentry-cli sourcemaps upload --org hep-vaud --project crt-backend ./dist"
```

Comment:
Are you sure you want to delete this? This effectively serves as documentation of where these files come from and how to update them..

Reply:
Mistake. Commit dropped.